### DISCUSSION TITLE
Why is there no isolines in Hong Kong?

### DISCUSSION THREAD

**Discussion by kf99916** [organicmaps/organicmaps#904] [Q&A] [answered]
As title. Thanks.

**Comment by rtsisyk** [comment, marked as answer]
Hi @kf99916, sorry for the late answer. This is definitely a bug. I filed a ticket: https://github.com/organicmaps/organicmaps/issues/1045

I hope that HikingBook is doing well. Please contribute fixes back to our upstream project, if any. Please drop an email to hello@organicmaps.app if you want to talk.

**Reply by kf99916** [reply]
Hi @rtsisyk , thank you for your reply and I have dropped an email to you.
